filer/private/sass/components/_drag-and-drop.scss

```diff
@@ -2,6 +2,9 @@
 // DRAG AND DROP
 
 form .form-row {
+    .related-widget-wrapper-link {
```

Comment:
can it affect other fields in the form like this that actually require related-widget stuff?